tests/end-to-end/external-solution-integration/deploy.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: e2e
+  namespace: production
+  labels:
+    app: e2e
+  annotations:
+    sidecar.istio.io/inject: "true"
+spec:
+  restartPolicy: Never
+  containers:
+  - name: e2e
+    image: k15r/e2e:bla
```

Comment:
Please put the image to Kyma google container registry

Reply:
it would be handled here: https://github.com/kyma-project/test-infra/issues/839